typescript/sdk/src/providers/ProviderType.ts

```diff
@@ -203,6 +203,23 @@ export interface EthersV5Transaction
   transaction: EV5Transaction;
 }
 
+export function createAnnotatedEthersV5Transaction(params: {
+  annotation: string;
+  chainId: number;
+  to: string;
+  data: string;
+}): Annotated<EthersV5Transaction> {
+  return {
+    annotation: params.annotation,
+    type: ProviderType.EthersV5,
+    transaction: {
+      chainId: params.chainId,
+      to: params.to,
+      data: params.data,
+    },
+  };
+}
```

Comment:
> im surprised this type is not an extension of EthersV5Transaction

do you mean something like this?
```ts
export interface AnnotatedEthersV5Transaction extends EthersV5Transaction {
  annotation: string;
}
```

basically just syntactic sugar I guess, alternatively could add something like this
```ts
export type AnnotatedEthersV5Transaction = Annotated<EthersV5Transaction>;
```